config/default.json

```diff
@@ -0,0 +1,3 @@
+{
+  "mongoURI": "mongodb+srv://RootUser:MJo5hRCZOX2Ui3Q3@movie.7h040wk.mongodb.net/?retryWrites=true&w=majority"
```

Comment:
Do not push sensitive data to gitHub. You can use dotenv for it.

Reply:
Thanks for the comment.
Added the dotenv package. Now the new .env file contains the URL for the database and is located in .gitignore

Reply:
Nice!